test-apps/imodel-transformer/package.json

```diff
@@ -25,7 +26,8 @@
     "@bentley/itwin-client": "2.17.0-dev.9",
     "fs-extra": "^8.1.0",
     "request-promise-native": "^1.0.5",
-    "yargs": "^16.0.0"
+    "yargs": "^16.0.0",
+    "xmldom": "^0.5.0"
```

Comment:
Minor nit - I like to alphabetize dependencies. If you don't, then you have to read the entire list when looking for a particular dependency.

Reply:
yeah I used `rush add` to add this dep and iirc it doesn't alphabetize, will fix